packages/strapi-admin/admin/src/components/Inputs/index.js

```diff
@@ -0,0 +1,129 @@
+/**
+ *
+ * Inputs
+ */
+
+import React from 'react';
```

Comment:
Why don't you use the `Inputs` from `@buffetjs/custom` so we can (if needed) update the codebase more easily?

You will just need to pass your `HeadersInput` and `EventInput` inputs to the component it will also work.